src/main/java/com/example/demo/repository/PatientRepository.java

```diff
@@ -60,6 +61,15 @@ public Patient updatePatient(String email, Patient newPatientData) {
         return getPatient(newPatientData.getEmail());
     }
 
+    public Patient updatePatientPassword(String email, String password) {
+        Patient patientToUpdate = getPatientReference(email);
+        if (password == null) {
```

Comment:
moim zdaniem ta walidacja mogtlaby byc na poczatku po co najpierw pobierac a potem srpawdzac -> w sytuacji jak nie mamy bazy danyhc jest to jeszcze do przyjecia natomiast jak mielibysmy baze danych to juz wtedy niestety marnowalibysmy niepotrzebnie połączenie